ovh-api-bash-client.sh

```diff
@@ -201,28 +229,70 @@ getJSONFieldString()
 {
     JSON="$1"
     FIELD="$2"
-    RESULT=$(echo $JSON | $CURRENT_PATH/$LIBS/JSON.sh | grep "\[\"$FIELD\"\]" | sed -r "s/\[\"$FIELD\"\]\s+(.*)/\1/")
+    RESULT=$(echo $JSON | $BASE_PATH/$LIBS/JSON.sh | grep "\[\"$FIELD\"\]" | sed -r "s/\[\"$FIELD\"\]\s+(.*)/\1/")
     echo ${RESULT:1:${#RESULT}-2}
 }
 
+# override CURRENT_PATH with profile name
+# usage : initProfile profile_name
+initProfile()
+{
+  local profile=$1
+  if [ -n "${profile}" ]; then
+
+    if [ ! -d "${PROFILES_PATH}/${profile}" ]
+    then
+     echo "${PROFILES_PATH}/${profile} should exists"
+     exit 1
+    fi
+    # override default configuration location
+    CURRENT_PATH="$( cd "${PROFILES_PATH}/${profile}" && pwd )"
+    HELP_CMD="${HELP_CMD} --profile ${profile}"
+  else
+    echo "profile not set, choose in : "
```

Comment:
In this case maybe we could use a `default` profile, to avoid these modifications to be breaking for existing users?